libmscore/element.h

```diff
@@ -127,7 +127,7 @@ class EditData {
       bool vRaster                     { false };
 
       int key                          { 0     };
-      Qt::KeyboardModifiers modifiers  { 0     };
+      Qt::KeyboardModifiers modifiers; // '0' initialized via default constructor
```

Comment:
Does this produce any warnings? It doesn't seem that any `QFlags` constructor got deprecated.

Reply:
Yes it does produce warnings, 383 in total, the vast majority of the ones I tried to fix. I don't think the constructor is deprecated, but the explicit initialization

Reply:
As mentioned [here](https://lists.qt-project.org/pipermail/development/2017-January/028489.html), another alternative could be using something like
```
Qt::KeyboardModifiers modifiers {};
```
or 
```
Qt::KeyboardModifiers modifiers = {};
```

Maybe some of the options above could be used to avoid both warnings and comment about the default constructor?

Reply:
Maybe... I see whether I can try out those suggestions, later...

Reply:
Very starnge, now I can't build using Qt 5.15.0 (snapshot) at all, tons og compiler errors reg. `QMultiMap<K,V>::insert`

Reply:
Could you post some of these errors here? Maybe they appeared after #5264, it uses `QMultiMap` for implementation of `ChangeMap` introduced in that PR.

Reply:
Darn, I've just change my build back to Qt 5.12.7...

Reply:
OK, here we go (sorry, they're in German)

    C:\Qt\5.15.0\msvc2017_64\include\QtCore/qmap.h(1213): error C2244: "QMultiMap<K,V>::insert": Keine Übereinstimmung für Funktionsdefinition mit vorhandener Deklaration gefunden [C:\Users\Jojo\Documents\GitHub\MuseScore\msvc.build_x64\audiofile\audiofile.vcxproj]
   C:\Qt\5.15.0\msvc2017_64\include\QtCore/qmap.h(1211): note: Siehe Deklaration
   von "QMultiMap<K,V>::insert"
   C:\Qt\5.15.0\msvc2017_64\include\QtCore/qmap.h(1213): note: Definition
   C:\Qt\5.15.0\msvc2017_64\include\QtCore/qmap.h(1213): note: 'QMultiMap<K,V>::
  iterator QMultiMap<K,V>::insert(const Key &,const T &)'
   C:\Qt\5.15.0\msvc2017_64\include\QtCore/qmap.h(1213): note: Vorhandene Deklarationen
   C:\Qt\5.15.0\msvc2017_64\include\QtCore/qmap.h(1213): note: 'QMap<K,V>::iterator QMultiMap<K,V>::insert(QMap<K,V>::const_iterator,const Key &,const T &)'
  C:\Qt\5.15.0\msvc2017_64\include\QtCore/qmap.h(1213): note: 'QMap<K,V>::itera
  tor QMultiMap<K,V>::insert(const Key &,const T &)'

Reply:
Maybe adding `#include <QMultiMap>` to `all.h` would help?

Reply:
Even though that seems to be the RightThing anyway, it doesn't help

Reply:
@jthistle: Mind to check?

Reply:
Then it looks like it is something to fix either for Qt or compiler developers. In Qt 5.15 implementation of `QMultiMap::insert()` seems to have changed as a result of https://github.com/qt/qtbase/commit/4ec6748c6a30f74e6d8fbb90fc118b306d1fa690. Now [declaration](https://github.com/qt/qtbase/blob/5.15/src/corelib/tools/qmap.h#L1120) of `QMultiMap::insert()` declares simply `iterator` as its return type (so it should probably be `QMultiMap::iterator`?) which is [declared](https://github.com/qt/qtbase/blob/5.15/src/corelib/tools/qmap.h#L1115) to be the same as `QMap::iterator`. The [definition](
https://github.com/qt/qtbase/blob/5.15/src/corelib/tools/qmap.h#L1211) explicitly refers to `QMultiMap::iterator` which should probably be correct, but for some reason your compiler considers it to be something different from the `iterator` used in that function declaration. I am not sure whether the code author or compiler is correct here.

Reply:
Waiting for Qt 5.15.0 Beta, in the vague hope that might fix this

Reply:
Qt 5.15.0 Aplha does not fix this :-(

Reply:
Now reported in https://bugreports.qt.io/browse/QTBUG-82166

Reply:
It does build with MSVC 2019!

Reply:
`Qt::KeyboardModifiers modifiers {};` works BTW

Reply:
It does build with MinGW too.